README.md

```diff
@@ -10,3 +10,16 @@ embedded in UOR artifacts.
 
 To learn more about Universal Runtime visit the [UOR-Framework](https://github.com/uor-framework/uor-framework) repository.
```

Comment:
We should point them at the end user site, and not the repository containing the site content